rockpaperscissors.py

```diff
@@ -77,126 +75,3 @@ def r3():
     r0()
 else:
     print('invalid entry, run script again')
-'''
-#version 2.0 May 5, 2020
```

Comment:
NOOOOOOO!!!!!!

Reply:
And this is just me explaining we are adding the other file in the next commit